plugins/plugin-chart-echarts/src/components/Echart.tsx

```diff
@@ -26,38 +26,54 @@ const Styles = styled.div<EchartsStylesProps>`
   width: ${({ width }) => width};
 `;
 
+const usePrevious = <T extends unknown>(value: T): T | undefined => {
+  const ref = useRef<T>();
+  useEffect(() => {
+    ref.current = value;
+  });
```

Comment:
Wouldn't this only store the initial value?

Reply:
I picked up this from StackOverflow and didn't look too closely at the details - I pulled this out and replaced with a simple `useRef` instead